test/unitTests/logging/OmnisharpChannelObserver.test.ts

```diff
@@ -21,16 +20,4 @@ suite("OmnisharpChannelObserver", () => {
         observer.post(event);
         expect(hasShown).to.be.true;
     });
-
```

Comment:
Why did we remove the test here?

Reply:
This test was testing the deprecated column functionality. Since we removed that parameter, we don't need to test for it.